ruby/codeql-extractor.yml

```diff
@@ -11,8 +11,21 @@ file_types:
   - name: ruby
     display_name: Ruby files
     extensions:
-       - .rb
+      - .rb
   - name: erb
     display_name: Ruby templates
     extensions:
-       - .erb
+      - .erb
+options:
+  trap:
+    title: Options related to TRAP output.
```

Comment:
May as well use the same title [as for C#](https://github.com/github/codeql/blob/main/csharp/codeql-extractor.yml#L19) (or change the title for C#, if you prefer this one).